dspace-rest/src/main/java/org/dspace/rest/CollectionsResource.java

```diff
@@ -97,8 +97,8 @@
     @Path("/{collection_id}")
     @Produces({ MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML })
     public org.dspace.rest.common.Collection getCollection(@PathParam("collection_id") String collectionId,
-            @QueryParam("expand") String expand, @QueryParam("limit") @DefaultValue("100") Integer limit,
-            @QueryParam("offset") @DefaultValue("0") Integer offset, @QueryParam("userIP") String user_ip,
+            @QueryParam("expand") String expand, @QueryParam("itemsLimit") @DefaultValue("100") Integer itemsLimit,
+            @QueryParam("itemsOffset") @DefaultValue("0") Integer itemsOffset, @QueryParam("userIP") String user_ip,
```

Comment:
My only concern is that changing this @QueryParam, we would be renaming /collections/{ID}?limit=10&offset=10 to /collections/{ID?itemsLimit=10&itemsOffset=10, causing an incompatibility.

So, perhaps it is fine to change the javadoc, and the variable name, but leave the existing QueryParam value.